mrblib/mruby-cli/setup.rb

```diff
@@ -386,11 +393,43 @@ def clean_env(envs)
   sh "rake deep_clean"
 end
 
+desc "generate a release tarball"
+task :release => :compile do
```

Comment:
I think it'll be fine. This PR is a refactoring of the version stuff, but with a fallback if we can't detect the version number I think we'll be good.